src_c/simd_blitters_sse2.c

```diff
@@ -490,21 +479,25 @@ alphablit_alpha_sse2_argb_no_surf_alpha_opaque_dst(SDL_BlitInfo *info)
                     /* src(ARGB) -> src1 (00000000ARGBARGB) */
                     LOAD_64_INTO_M128(srcp64, &src1);
 
-                    /* created squashed alpha -> mm_src_alpha
-                     * (0000000000000A0A) */
+                    /* isolate alpha channels
+                     * 00000000A1000A2000 -> mm_src_alpha */
+                    mm_src_alpha = _mm_andnot_si128(mm_rgb_mask, src1);
+
+                    /* shift right to position alpha channels for manipulation
+                     * 000000000A1000A200 -> mm_src_alpha*/
+                    mm_src_alpha = _mm_srli_si128(mm_src_alpha, 1);
+
+                    /* shuffle alpha channels to duplicate 16 byte pairs
+                     * 000000000A10A10A20A2 -> mm_src_alpha */
```

Comment:
```suggestion
                   /* shuffle alpha channels to duplicate 16 byte pairs
                     * shuffle operation on lower half = (3,3,1,1)
                     * 000000000A10A10A20A2 -> mm_src_alpha */
```

Another suggestion to go with the suggestion above, for those of us whose brains can't digest binary easily.